shell/browser/api/electron_api_web_contents.h

```diff
@@ -45,12 +45,7 @@
 #if BUILDFLAG(ENABLE_PRINTING)
 #include "chrome/browser/printing/print_view_manager_basic.h"
 #include "components/printing/common/print_messages.h"
-#include "printing/backend/print_backend.h"  // nogncheck
 #include "shell/browser/printing/print_preview_message_handler.h"
-
-#if defined(OS_WIN)
-#include "printing/backend/win_helper.h"
```

Comment:
Seems like we are using some classes from this header on windows.

Reply:
yep, just wanted to know which ones, so i asked CI :)